definitions/npm/react-router-dom_v4.x.x/flow_v0.53.x-v0.62.x/react-router-dom_v4.x.x.js

```diff
@@ -66,8 +66,8 @@ declare module "react-router-dom" {
     goForward(): void,
     canGo?: (n: number) => boolean,
     block(
```

Comment:
Looks like `block` should also allows strings? : `string | callback` https://github.com/ReactTraining/history#blocking-transitions

Reply:
Will fix.